conf/.claude/settings.json

```diff
@@ -5,14 +5,14 @@
       "mcp__slack__channels_list",
       "mcp__slack__conversations_search_messages",
       "mcp__slack__conversations_add_message",
-      "mcp__gitlab__list_project_merge_requests",
+      "mcp__gitlab__list_merge_requests",
```

Comment:
Do we need something else for the tags here? or this actions are enough? did you check on your setup what is the content of the settings?

Otherwise LGTM